images/atomic-openshift-node-problem-detector.yml

```diff
@@ -1,14 +1,14 @@
 container_yaml:
   go:
     modules:
-    - module: github.com/openshift/node-problem-detector
+    - module: github.com/openshift-priv/node-problem-detector
```

Comment:
we don't want the go module changed. the go module continues to be the public repo.